src/compiler/resolver.cc

```diff
@@ -1594,7 +1642,8 @@ void Resolver::fill_classes_with_skeletons(std::vector<Module*> modules) {
           auto method = member->as_Method();
           auto method_is_abstract = method->is_abstract();
 
-          check_method(method, ir_class, &member_name, &kind);
+          bool allow_future_reserved;
+          check_method(method, ir_class, &member_name, &kind, allow_future_reserved = false);
```

Comment:
I prefer assigning to the local in the previous line (like you do in most other places).